spec/bolt/cli_spec.rb

```diff
@@ -224,39 +238,21 @@ def stub_config(file_content = {})
     end
 
     context 'install' do
-      let(:flags)       { %W[--project #{project}] }
-      let(:cli)         { Bolt::CLI.new(%w[module install] + flags) }
-      let(:config_data) { { 'modules' => [{ 'name' => 'puppetlabs-yaml' }] } }
+      let(:command)        { %W[module install --project #{project}] }
+      let(:project_config) { { 'modules' => [] } }
 
-      it 'errors if missing a bolt-project.yaml' do
-        FileUtils.rm(config)
-        expect { cli.execute(cli.parse) }.to raise_error(
-          Bolt::Error,
-          /Could not find project configuration file/
-        )
-      end
-
-      it 'exits successfully if there are no project modules' do
-        File.write(config, {}.to_yaml)
-        expect(installer).not_to receive(:install)
+      it 'does nothing if project config has no module declarations' do
+        allow(project).to receive(:modules).and_return([])
         result = cli.execute(cli.parse)
         expect(result).to eq(0)
+        expect((project_path + 'Puppetfile').exist?).to eq(false)
+        expect((project_path + '.modules').exist?).to eq(false)
       end
 
-      it 'installs project modules' do
+      it 'runs' do
         expect(installer).to receive(:install)
         cli.execute(cli.parse)
       end
-
-      it 'installs project modules forcibly' do
-        cli = Bolt::CLI.new(%W[module install --project #{project} --force])
```

Comment:
And here.

Reply:
This is covered by this test: https://github.com/puppetlabs/bolt/blob/main/spec/bolt/module_installer_spec.rb#L87